src/js/containers/covid19/recipient/RecipientContainer.jsx

```diff
@@ -20,15 +20,15 @@ const RecipientContainer = () => {
         <div className="body__content recipient__container">
             <DateNote />
             <h3 className="body__narrative">
-                <strong>Who</strong> received funding through COVID-19 Response awards?
+                <strong>Who</strong> received funding through COVID-19 awards?
             </h3>
             <div className="body__narrative-description">
                 <p>
                     Once the federal government has determined that an individual, organization, business, or state, local, or tribal government will receive an award, the money is obligated (promised) and then outlayed (paid) according to the terms of the contract or financial assistance.<sup>3</sup>
                 </p>
                 <ReadMore>
                     <p>
-                        In the map below, you will see a spending breakdown by state, county, or congressional district of recipients who have received awards funded by COVID-19 appropriations. These breakdowns are based on recipients’ addresses. In the case of individual recipients, data is aggregated by county or state to protect personally identifiable information (PII). Data about the location where the award money is used, known as the ‘Primary Place of Performance’, is available in the profile page <strong>download</strong>.
+                        In the map below, you will see a spending breakdown by state, county, or congressional district of recipients who have received awards funded by COVID-19 appropriations. These recipient locations shown below reflect the location of organizational offices that were used to register for awards, and do not reflect the actual distribution of funds. In the case of recipients who are individual persons and not organizations, data is aggregated by county or state to protect personally identifiable information (PII). Data about the location where the award money is used, known as the &apos;Primary Place of Performance&apos;, is available in the download at the top right of this page.
```

Comment:
```suggestion
                        In the map below, you will see a spending breakdown by state, county, or congressional district of recipients who have received awards funded by COVID-19 appropriations. These recipient locations shown below reflect the location of organizational offices that were used to register for awards, and do not reflect the actual distribution of funds. In the case of recipients who are individual persons and not organizations, data is aggregated by county or state to protect personally identifiable information (PII). Data about the location where the award money is used, known as the &apos;Primary Place of Performance&apos;, is available in the download at the top right of the page.
```
If we want to exactly match the doc

Reply:
Thank you for catching this!! Let's keep "this" as that wording is directly from a message from Andrew